backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/init/CassandraTypesCreatorTest.java

```diff
@@ -52,24 +53,28 @@ void setUp(CassandraCluster cassandra) {
     @Test
     void getDefinedUserTypeShouldNotReturnNullNorFailWhenTypeIsDefined() {
         assertThat(cassandra.getTypesProvider().getDefinedUserType(TYPE_NAME))
-            .isNotNull();
+                .isNotNull();
     }
 
     @Test
     void initializeTypesShouldCreateTheTypes() {
         cassandra.getConf().execute(SchemaBuilder.dropType(TYPE_NAME));
 
-        new CassandraTypesCreator(MODULE, cassandra.getConf()).initializeTypes();
+        assertThat(new CassandraTypesCreator(MODULE, cassandra.getConf()).initializeTypes())
+                .isTrue();
+
         CassandraTypesProvider cassandraTypesProviderTest = new CassandraTypesProvider(MODULE, cassandra.getConf());
         assertThat(cassandraTypesProviderTest.getDefinedUserType(TYPE_NAME))
-            .isNotNull();
+                .isNotNull();
     }
 
     @Test
     void initializeTypesShouldNotFailIfCalledTwice() {
-        new CassandraTypesProvider(MODULE, cassandra.getConf());
+        assertThat(new CassandraTypesCreator(MODULE, cassandra.getConf()).initializeTypes())
```

Comment:
I don't agree, the first line is environment setup, you don't have to assert on it

Reply:
I disagree, I check that if *trying* to set up the environment again, it tells that nothing have been done.

Reply:
oh sorry, I missed the `initializeTypes` call.
Still, your description sounds like a new test case

Reply:
I do not have idea to improve it actually :/

Reply:
split that in two tests

Reply:
Ok, maybe I don't understand but here is what I read from this test.
Test name `initializeTypesShouldNotFailIfCalledTwice` explains that calling `initializeTypes` twice should not fail.
I expect this method to be called twice and the assertion would be "I don't expect it to throw".
Unfortunately, the previous version of the test was really poor at making this explicit but it's basically what it does.
Now, you replaced the implicit stuff with another call to `initializeTypes`, that's great.
But the assertion you added checks something new: it checks that the second call to `initializeTypes` return false.
You are now asserting that:
1. it doesn't fail
2. it does not apply any change

This second check belongs to a new test method that could be named `initializeTypesShouldReturnFalseIfCalledTwice`

Are you ok with that ?

Reply:
Ok, I understand now, thanks.